node/resolvers/utils/checkConfig.ts

```diff
@@ -345,7 +352,10 @@ export const checkConfig = async (ctx: Context) => {
     return null
   }
 
-  if (!settings?.adminSetup?.cartLifeSpan) {
+  if (
```

Comment:
Shouldn't this check be an OR instead of an AND?

Reply:
This is a check to return to the default settings if no setting is filled in yet. Therefore, it has to be AND.